File: assignment 10 - C# practice problems/js/29.js
//Write a JavaScript function to convert a positive number to negative number. 
function positiveToNegaive(numberParam) {
    return -numberParam;
}

function positiveToNegaiveDriverFunction() {
    let numberInput = prompt("Enter a positive Number");
    alert("The negative is " + positiveToNegaive(parseFloat(numberInput)));
}

positiveToNegaiveDriverFunction()

File: assignment 10 - C# practice problems/29 .cs
//Write a c# function to convert a positive number to negative number. 

using System;
namespace MyNameSpace
{
    class Program29
    {
        public static void PositiveToNegativeNumber()
        {

            Console.WriteLine("Enter a negative Number: ");
            var numberInput = decimal.Parse(Console.ReadLine());
            Console.WriteLine("Positive number is: " + Math.Abs(numberInput));


        }

    }
}
